pydra/engine/tests/test_task.py

```diff
@@ -499,6 +547,236 @@ def no_annots(c, d):
     assert result.output.out == (20.2, 13.8)
 
 
+def test_input_spec_func_1(use_validator):
+    """ the function w/o annotated, but input_spec is used """
+
+    @mark.task
+    def testfunc(a):
+        return a
+
+    my_input_spec = SpecInfo(
+        name="Input",
+        fields=[("a", attr.ib(type=float, metadata={"help_string": "input a"}))],
+        bases=(FunctionSpec,),
+    )
+
+    funky = testfunc(a=3.5, input_spec=my_input_spec)
```

Comment:
i like this and perhaps extend this to a potential helper function (please see this for the kind of boilerplate i would like to reduce: https://github.com/nipype/pydra-ml/blob/master/pydra_ml/classifier.py#L25

`funky = to_task(function, optional input_spec)`

what if `testfunc` was already an annotated function. would input_spec override the annotation?

Reply:
yes, it will overwrite the function annotation, see: https://github.com/nipype/pydra/pull/332/files#diff-7c80889b6dbc9a56fcd4821c277e3e95R623

Reply:
your example of boilerplate was for output, and this is for input right now, perhaps I could add an `output_spec` for `FunctionTask` in this PR. 
But you would like to have a shorter version of `input_spec` or you want to have really a wrapper that adds annotation to a python function?